config/environments/production.rb

```diff
@@ -93,7 +93,7 @@
 
   # Use default logging formatter so that PID and timestamp are not suppressed.
   config.log_formatter = ::Logger::Formatter.new
-  config.assets.precompile = ['*.js', '*.css', '*.css.erb']   
+  config.assets.precompile = ['*.js', '*.css', '*.css.erb']
   config.assets.compile = true
-  PUBLIC_URL = "fsektionen.se"  
+  PUBLIC_URL = "fsektionen.se"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.